src/read_vegparam.c

```diff
@@ -239,16 +239,23 @@ veg_con_struct *read_vegparam(FILE *vegparam,
         nrerror(ErrStr);
       }
 
+      for ( j = 0; j < 12; j++ ) {
```

Comment:
Answer - yes it is your job.  This will be merged into develop and if its not your job, it quickly becomes mine :).  Just change this one location.